### PR TITLE
test : Complete test cases for the (current-version) protocol by MessageType

### LINKED ISSUE #6213: Test examples from the current release to support communication-related coverage tests
## Why you need it?
 Is your feature request related to a problem? Please describe in details
添加当前版本的测试例子，来支持通信相关的覆盖测试
Test examples from the current release to support communication-related coverage tests

## How it could be?
A clear and concise description of what you want to happen. You can explain more about input of the feature, and output of it.

### PR DESCRIPTION
**Pull request by Bughue**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
Complete test cases for the (current-version) protocol by MessageType.  This also provides the basis for comprehensive testing of multiple versions of the protocol.

### Ⅱ. Does this pull request fix one issue?

fixes https://github.com/apache/incubator-seata/issues/6213

### CODE REVIEW

File: test/src/test/java/io/seata/core/rpc/netty/ChannelManagerTestHelper.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
```
Comment (xingfudeshi) [inline test/src/test/java/io/seata/core/rpc/netty/ChannelManagerTestHelper.java]: Please remove seata copyright

File: test/src/test/java/io/seata/core/rpc/netty/ChannelManagerTestHelper.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import io.netty.channel.Channel;
+import io.seata.core.rpc.netty.current.ProtocolTestConstants;
+
+import java.util.concurrent.ConcurrentMap;
+
+/**
+ * Channel Manager Test Helper
+ *
+ * @author minghua.xie
+ * @date 2023/12/25
```
Comment (xingfudeshi) [inline test/src/test/java/io/seata/core/rpc/netty/ChannelManagerTestHelper.java]: remove @author and @date

**Comment by xingfudeshi** [comment]
1.Remove the seata copyright information
2.Remove @author @date
3.Rgister this PR in [2.x](https://github.com/apache/incubator-seata/blob/2.x/changes/zh-cn/2.x.md) and [2.x](https://github.com/apache/incubator-seata/blob/2.x/changes/en-us/2.x.md)

**Comment by Bughue** [comment]
建议把mockserver合完再合这个 https://github.com/apache/incubator-seata/pull/6205

**Pull request closed without merging**